### PR TITLE
Np gene metrics remove cols from optimus and slideseq

### PR DESCRIPTION
**Pull request by nikellepetrillo**
### Description

Our gene metrics.csv produced by Optimus/SlideSeq has empty or incorrect counts for exonic reads, intronic reads, unmapped reads, intergenic reads and duplicate reads. We need to remove these columns. This bug is the result of sctools trying to grab these counts from the XF tag of the BAM file produced by STARsolo (the XF tag does not exist).

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [x] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

**Comment by nikellepetrillo** [comment]
Optimus and SlideSeq are failing in CompareCellMetrics and CompareGeneMetrics. This is expected as we are removing columns from the cell metrics/gene metrics csv files. Here is a google sheet showing a side by side comparison of the files:

https://docs.google.com/spreadsheets/d/17ceoJs5NA4ypO56XWnX3AdBO5ixgstfTj4Gx8cZMbo4/edit#gid=1949303814

File: pipelines/skylab/slideseq/SlideSeq.changelog.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 1.0.3
+2023-03-27 (Date of Last Commit)
+* Removed the following columns from the gene metrics csv and the Loom as the counts were empty/incorrect: reads_unmapped, reads_mapped_exonic, reads_mapped_intronic, reads_mapped_utr, reads_mapped_intergenic, duplicate_reads. We also removed duplicate_reads from the cell metrics csv and the Loom.
```
Comment (ekiernan) [inline pipelines/skylab/slideseq/SlideSeq.changelog.md:3]:
```suggestion
* Removed the following columns from the gene metrics csv and the Loom as the counts were empty/incorrect: reads_unmapped, reads_mapped_exonic, reads_mapped_intronic, reads_mapped_utr, reads_mapped_intergenic, duplicate_reads. We also removed duplicate_reads from the cell metrics csv and the Loom

File: pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.changelog.md [hunk 2]
```diff
@@ -1,4 +1,9 @@
-# 1.2.19
+# 1.2.20
+2023-03-27 (Date of Last Commit)
+
+* SlideSeq-specific and Optimus-specific changes to Metrics.wdl. This change does not affect the MultiSampleSmartSeq2SingleNucleus pipeline.
```
Comment (ekiernan) [inline pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.changelog.md:4]:
```suggestion
* SlideSeq-specific and Optimus-specific changes to Metrics.wdl. This change does not affect the MultiSampleSmartSeq2SingleNucleus pipeline
```

File: pipelines/skylab/smartseq2_single_sample/SmartSeq2SingleSample.changelog.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+# 5.1.19
+2023-03-27 (Date of Last Commit)
+
+* SlideSeq-specific and Optimus-specific changes to Metrics.wdl. This change does not affect the SmartSeq2SingleSample pipeline.
```
Comment (ekiernan) [inline pipelines/skylab/smartseq2_single_sample/SmartSeq2SingleSample.changelog.md:4]:
```suggestion
* SlideSeq-specific and Optimus-specific changes to Metrics.wdl. This change does not affect the SmartSeq2SingleSample pipeline

**Review by kevinpalis — APPROVED** [review]
LGTM!

**Pull request merged**